src/app_engine/gcm_config_DO_NOT_CHECK_IN.json

```diff
@@ -0,0 +1,3 @@
+{
+  "GCM_API_KEY": ""
```

Comment:
What about not checking in this file? Just show an example of the file in
README.
We could also modify the build script to generate the temple.

On Fri, Feb 6, 2015 at 11:50 AM, Zeke Chin <notifications@github.com> wrote:

> In src/app_engine/gcm_config_DO_NOT_CHECK_IN.json
> <https://github.com/webrtc/apprtc/pull/40#discussion_r24266785>:
>
> > @@ -0,0 +1,3 @@
> > +{
> > +  "GCM_API_KEY": ""
>
> can't git ignore a file that's already tracked (ie committed). It'll still
> show up if you edit it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/40/files#r24266785>.
>



-- 
Jiayang Liu